openshift/e2e-common.sh

```diff
@@ -495,6 +496,21 @@ function run_e2e_tests(){
     --skip-cleanup-on-fail \
     --resolvabledomain || failed=1
 
+
+  # Allow to use any seccompProfile for non default cases,
+  # for more check https://docs.openshift.com/container-platform/4.12/authentication/managing-security-context-constraints.html
+  oc adm policy add-scc-to-user privileged -z default -n serving-tests
```

Comment:
Just out of curiosity, why do we need that here?

Reply:
The tests partially check for the unsafe scenarios. In order to set a value other than the default for seccompProfile you need to have a scc setup. I could setup one like but privileges is by default there and allows any seccompProfile to be set. Alternatively I could have:

```
kind: SecurityContextConstraints
apiVersion: security.openshift.io/v1
metadata:
  name: scc-test
runAsUser:
  type: MustRunAsRange
seLinuxContext:
  type: MustRunAs
seccompProfiles: ["Unconfined"]

oc adm policy add-scc-to-user  scc-test -z default -n serving-tests
```